codegen/projections/rails_json/lib/rails_json/types.rb

```diff
@@ -76,52 +76,59 @@ module Types
     #   @return [Array<Integer>]
     # @!attribute query_params_map_of_string_list
     #   @return [Hash<String, Array<String>>]
-    AllQueryStringTypesInput = ::Struct.new(
-      :query_string,
-      :query_string_list,
-      :query_string_set,
-      :query_byte,
-      :query_short,
-      :query_integer,
-      :query_integer_list,
-      :query_integer_set,
-      :query_long,
-      :query_float,
-      :query_double,
-      :query_double_list,
-      :query_boolean,
-      :query_boolean_list,
-      :query_timestamp,
-      :query_timestamp_list,
-      :query_enum,
-      :query_enum_list,
-      :query_integer_enum,
-      :query_integer_enum_list,
-      :query_params_map_of_string_list,
-      keyword_init: true
-    ) do
+    class AllQueryStringTypesInput
       include Hearth::Structure
+
+      MEMBERS = %i[
```

Comment:
What about having Hearth::Structure define a class method for `members` which would then setup the attr accessors (and set the MEMBERS).   Eg:

```
class MyType
  include Hearth::Structure

  members %i(member1 member2)
end

Reply:
I was avoiding adding new methods because then those have to be reserved. Class method may be ok  too.